docs/duration.md

```diff
@@ -30,6 +30,93 @@ For more detailed information, see the ISO 8601 standard or the [Wikipedia page]
 | **PT0S**             | Zero |
 | **P0D**              | Zero |
 
+### **Kinds of Durations and Daylight Saving Time (DST)**
+
+There are two kinds of durations that can be stored in a `Temporal.Duration`:
+* An "absolute duration" counts real-world elapsed time between two instants, like a stopwatch does.  To get an absolute duration, use  the `difference()` method of `Temporal.Absolute`.
+* A "date/time duration" counts the difference in calendar days and local clock time. To get a date/time duration, use  the `difference()` method of `Temporal.DateTime`, `Temporal.Date`, `Temporal.Time`, or `Temporal.YearMonth` .
+
+Both kinds of durations usually have the same value. But if a DST transition (or other timezone-offset change) happened during the duration being measured, then the date/time duration will be smaller or larger than the real-world time reported by an absolute duration. This occasional difference can lead to subtle bugs in your code, like charging a customer for a 2-day rental on the 25-hour day when DST ends in the Fall.
+
+To avoid DST-related bugs when using `Temporal.Duration`, follow three best practices: 
+
+1. **Don't assume that every day is 24 hours long.**<br/>
+For example, don't add 24 hours to a `Temporal.Absolute` and expect that the result will exactly be one day later in local time. To add full calendar days, first convert absolute times to a `Temporal.DateTime`, then add days, and finally convert back to `Temporal.Absolute`. When working with absolute times or absolute durations, rely on `Temporal` to convert hours into days (or vice versa) instead of multiplying or dividing by 24 in your code.
+```js
+const dt = new Temporal.DateTime(2020, 11, 1); // midnight on the day DST starts
```

Comment:
In other docs/examples people have suggested not to use DST changes occurring in the future, since things may change between now and that date, making the documentation stale.

Reply:
I agree in general, but I think 5 months in the future should be fine, especially given that US time zone changes are communicated years in advance.  But happy to change it to a past date if you think that'd be better.

Reply:
I'll resolve this convo-- feel free to re-open if you want me to change this.